project/src/views/evaluations/evaluationsViews.js

```diff
@@ -1,45 +1,21 @@
 const EvaluationsPage = function(options = {}) {
-    const rows = [{
-        name: "Popescu Adrian",
-        technology: "Javascript",
-        level: "Mid2"
-    },
-    {
-        name: "Popescu Adrian",
-        technology: "Javascript",
-        level: "Mid2"
-    },
-    {
-        name: "Popescu Adrian",
-        technology: "Javascript",
-        level: "Mid2"
-    },
-    {
-        name: "Popescu Adrian",
-        technology: "Javascript",
-        level: "Mid2"
-    }];
-    const headings = ['Nume', 'Tehnologie', 'Nivel', ''];
     return `
         ${NAV()}
         ${EvaluationsTable({
-            itemsHeadings: headings,
-            items: rows
+            itemsHeadings: getHeadings(),
```

Comment:
Would `headings` have sufficed instead of `itemsHeadings`?

Reply:
Yes, I've changed it to headings